hat/assets/js/apps/Iaso/utils/mapUtils.ts

```diff
@@ -206,3 +206,91 @@ export const polygonDrawOption = (
 
 export const getleafletGeoJson = (geoJson: any): void =>
     geoJson ? L.geoJson(geoJson, shapeOptions) : null;
+
+// TODO: this should be available in the new version of leaflet
+type LatLng = {
+    lat: number;
+    lng: number;
+};
+export type Bounds = {
+    _northEast: LatLng;
+    _southWest: LatLng;
+    // eslint-disable-next-line no-unused-vars
+    extend: (bounds: Bounds) => Bounds;
+};
+type BoundsOptions = {
+    padding: number[];
+};
+type DefaultViewPort = {
+    center: number[];
+    zoom: number;
+};
+type Map = {
+    leafletElement: {
+        // eslint-disable-next-line no-unused-vars
+        fitBounds: (bounds: Bounds, boundsOptions) => void;
+    };
+};
+
+export const getOrgUnitBounds = (orgUnit: OrgUnit): Bounds | undefined => {
```

Comment:
nitpick: `getSingleOrgUnitBounds` or `getOrgUnitsListBounds` for the other function would be better names in my opinion. Easier and quicker to tell the 2 func apart